src/config/texts.js

```diff
@@ -17,7 +17,14 @@ export const texts = {
       objectLoadErrorAlert:
         'Beim Laden des 3D-Objekts ist ein Problem aufgetreten. Bitte versuchen Sie es erneut.',
       screenRecordingCompleted: 'Die Aufnahme wurde erfolgreich in Ihrer Galerie gespeichert.',
-      screenRecordingError: 'Bei der Bildschirmaufzeichnung ist ein Fehler aufgetreten.'
+      viroRecordingError: {
+        0: 'Bei der Bildschirmaufzeichnung ist ein Fehler aufgetreten.',
+        1: 'Bitte erlauben Sie die Aufnahme von Videos/Screenshots.',
+        2: 'Bei der Initialisierung der Bildschirmaufzeichnung ist ein Fehler aufgetreten.',
+        3: 'Beim Speichern der Datei ist ein Fehler aufgetreten.',
+        4: 'Ihr System zeichnet bereits auf.',
+        5: 'Ihr System kann derzeit nicht aufzeichnen.'
```

Comment:
ℹ️ according to https://github.com/ViroCommunity/viro/blob/main/components/ViroConstants.ts#L14-L23